docs/github/getting-started.md

```diff
@@ -84,26 +82,26 @@ jobs:
           lfs: true
 
       # Cache
-      - uses: actions/cache@v1.1.0
+      - uses: actions/cache@v2.1.3
         with:
           path: Library
           key: Library
 
       # Test
       - name: Run tests
-        uses: webbertakken/unity-test-runner@v1.3
+        uses: webbertakken/unity-test-runner@v2
```

Comment:
We should probably also change from `webbertakken` to `game-ci`.

Reply:
I don't know much about github actions. Renaming a github actions repo does rename the action itself?

Reply:
Yea you can now use game-ci/action-name as they are moved to game-ci. I just published a new version of builder, which indeed referenced the same marketplace action.

Reply:
![image](https://user-images.githubusercontent.com/20756439/103692596-466b6a00-4f98-11eb-87f8-beb5c9e8e0b5.png)

Reply:
This change probably needs to be done at a few other places, not only in getting-started like I did. 👍